aivpn.py

```diff
@@ -55,160 +43,159 @@ def manage_info(REDIS_CLIENT,profile_name):
     """
-    try:
-        logging.debug(f'Provision OpenVPN: {identity}')
-
-        msg_id = 1
-        msg_request = "openvpn"
-        msg_addr = identity
-        msg_type = validate_identity(identity)
-
-        if msg_id and msg_request and msg_addr and msg_type:
-            # Add to privisioning queue
-            logging.debug(f"Adding item to provisioning queue. Msg ID: {msg_id}, msg_type: {msg_type}, msg_addr: {msg_addr}, msg_request: {msg_request}")
-            status = add_item_provisioning_queue(REDIS_CLIENT,msg_id,msg_type,msg_addr,msg_request)
-            redis_client.publish('services_status', 'MOD_CLI:NEW_REQUEST')
-            print(f"Provisioning triggered: {status}. Number of items in the queue: {list_items_provisioning_queue(REDIS_CLIENT)}")
-        else:
-            print('Provisioning process failed, try again')
+    identity_valid_data = get_validated_data(identity)
+    logging.debug(f'Provision No VPN: {identity}')
+    if identity_valid_data["msg_request"][0] and identity_valid_data["msg_type"]:
+        # Add to provisioning queue
+        status = add_item_provisioning_queue(REDIS_CLIENT, identity_valid_data["msg_id"], identity_valid_data["msg_type"],
+                                             identity_valid_data["msg_addr"], identity_valid_data["msg_request"][0])
+        redis_client.publish('services_status', 'MOD_COMM_RECV:NEW_REQUEST')
+        print(
+            f"Provisioning triggered: {status}. Number of items in the queue: {list_items_provisioning_queue(REDIS_CLIENT)}")
+    else:
+        print('Provisioning process failed in provision_new_novpn, try again')
 
-    except Exception as err:
-        print(f'Exception in provision_new_openvpn: {err}')
 
-def provision_wireguard(REDIS_CLIENT,identity):
+def provision_wireguard(REDIS_CLIENT, identity):
     """
     Trigger the provisioning of a new Wireguard profile for a client
     """
-    try:
-        logging.debug(f'Provision Wireguard: {identity}')
-        msg_id = 1
-        msg_request = "wireguard"
-        msg_addr = identity
-        msg_type = validate_identity(identity)
-
-        if msg_id and msg_request and msg_addr and msg_type:
-            # Add to privisioning queue
-            logging.debug(f"Adding item to provisioning queue. Msg ID: {msg_id}, msg_type: {msg_type}, msg_addr: {msg_addr}, msg_request: {msg_request}")
-            status = add_item_provisioning_queue(REDIS_CLIENT,msg_id,msg_type,msg_addr,msg_request)
-            redis_client.publish('services_status', 'MOD_COMM_RECV:NEW_REQUEST')
-            print(f"Provisioning triggered: {status}. Number of items in the queue: {list_items_provisioning_queue(REDIS_CLIENT)}")
-        else:
-            print('Provisioning process failed, try again')
-    except Exception as err:
-        print(f'Exception in provision_new_wireguard: {err}')
+    valid_data = get_validated_data(identity)
+    logging.debug(f'Provision No VPN: {identity}')
```

Comment:
Should be Wireguard not No VPN: `logging.debug(f'Provision Wireguard: {identity}')`